site/reference/schema/kptfile/README.md

```diff
@@ -0,0 +1,87 @@
+# `Kptfile`
+
+```yaml
+# API version of the Kptfile.
+apiVersion: kpt.dev/v1alpha2
+# Always "kptfile."
+kind: Kptfile
+# Metadata of the resource described by the kptfile.
+metadata:
+  # Name of this resource.
+  name: wordpress
+# The upstream locator for a package.
+upstream:
+  # The type of origin for a package. Only supports "git."
+  type: git
+  # The locator for a package on Git.
+  git:
+    # The repository of the package.
+    repo: https://github.com/GoogleContainerTools/kpt
+    # The sub directory in the git repository.
+    directory: /package-examples/wordpress
+    # A git branch, tag, or a commit SHA-1.
+    ref: v0.1
+  # The method used in updating from upstream.
+  # Supports "resource-merge," "fast-forward" and "force-delete-replace."
+  updateStrategy: resource-merge
+# Declares the pipeline of functions used to mutate or validate resources..
+pipeline:
+  # A list of of KRM functions that mutate resources.
+  mutators:
+    # Specifies the function container image.
+    # Can be fully qualified or infer the host path from
+    # kpt's configuration (defaults to gcr.io/kpt-fn)
+    - image: gcr.io/kpt-fn/apply-setters:v0.1
+    # An inline KRM resource used as the function config.
+    # Cannot be specified alongside config or configPath.
+    configMap:
```

Comment:
Here's the problem with providing an example. There are mutually exclusive fields such as `configMap` and `configPath`. We want the schema for both. I suggest rendering the actual schema instead of an example. 

In addition, maintaining this example by hand means that it will inevitable diverge from the schema.

@droot WDYT?

Reply:
Yeah, mutually exclusive fields makes this approach difficult. For now, @etefera Can you add another mutator function with `configPath` pointing to a file on the local disk.

I think driving this from single source of truth is a good idea, but may take a bit longer to get this in place. In the interim, this placeholder is needed so that rest of the docs/references can point to it. Let's improve it in `m4` milestone.

Reply:
Done.